build/config/linux/pkg_config.gni

```diff
@@ -83,6 +84,20 @@ if (host_pkg_config != "") {
   host_pkg_config_args = pkg_config_args
 }
 
+if (sysroot != "") {
+  pkg_config_args += [
+    "-s",
+    sysroot,
+  ]
+
+  if (pkgconfig_libdir != "") {
+    pkg_config_args += [
+      "--system_libdir",
+      pkgconfig_libdir,
```

Comment:
Please call it system_libdir and put it in the original location

https://source.chromium.org/chromium/chromium/src/+/main:build/config/linux/pkg_config.gni;drc=a8c8396fd20d98666d517c45b358c63736e345ef;l=47